.env.example

```diff
@@ -16,9 +16,11 @@ LINK_USD_PRICE_FEED='0x2c1d072e956AFFC0D435Cb7AC38EF18d24d9127c'
 MIN_WITHDRAW_AMOUNT='1'
 MAX_DEPOSIT_AMOUNT='100'
 MIN_BALANCE_PREMIUM='10'
+NONFUNGIBLE_POSITION_MANAGER='0xC36442b4a4522E871399CD717aBDD847Ab11FE88'
 STAGING_KEEPER_REGISTRY='0xE16Df59B887e3Caa439E0b29B42bA2e7976FD8b2'
 STAGING_KEEPER_REGISTRAR='0x57A4a13b35d25EE78e084168aBaC5ad360252467'
 STAGING_SWAP_ROUTER='0xE592427A0AEce92De3Edee1F18E0157C05861564'
-STAGING_LINK_TOKEN='0x326C977E6efc84E512bB9C30f76E30c160eD06FB'
-STAGING_PAYMENT_USD_PRICE_FEED='0x0d79df66BE487753B02D015Fb622DED7f0E9798d'
-STAGING_LINK_USD_PRICE_FEED='0x48731cF7e84dc94C5f84577882c14Be11a5B7456'
+STAGING_LINK_TOKEN='0xa36085F69e2889c224210F603D836748e7dC0088'
```

Comment:
Is this the right address?

Reply:
My bad, at some point i changed the price feeds and forgot to move the original ones.